integrations/optimizely/lib/index.js

```diff
@@ -121,7 +121,7 @@ Optimizely.prototype.track = function(track) {
   var optimizelyOptions = track.options('Optimizely');
   // Prefer the client instance specified via options, falling back to the globally defined one.
   var optimizelyClientInstance =
-    optimizelyOptions.clientInstance || window.optimizelyClientInstance;
+    optimizelyOptions.FSClientInstance || window.optimizelyClientInstance;
```

Comment:
For customers, would `FullStackClientInstance` be more helpful? I think `FS` makes more sense for me internally, but think the slightly longer, more verbose option couldn't hurt.